contributors/devel/sig-scalability/kubemark-guide.md

```diff
@@ -64,17 +64,17 @@ these comments with a clean API that would allow kubemark to run on top of any p
 To start a Kubemark cluster on GCE you need to create an external kubernetes
 cluster (it can be GCE, GKE or anything else) by yourself, make sure that kubeconfig
 points to it by default, build a kubernetes release (e.g. by running
-`make quick-release`) and run `test/kubemark/start-kubemark.sh` script.
+`make quick-release`) and run `kubernetes/test/kubemark/start-kubemark.sh` script.
```

Comment:
This isn't correct - if you're in the main k/k directory, the directory is test/kubemark/start-kubemark.sh (this is the directory from where you run make quick-release also).

Adding kubernetes/ to it would be misleading.